packages/integrations/src/download-client/sabnzbd/sabnzbd-schema.ts

```diff
@@ -87,7 +87,7 @@ export const historySchema = z.object({
         size: z.string(),
         category: z.string(),
         pp: z.string(),
-        retry: z.number(),
+        retry: z.number().or(z.boolean()),
```

Comment:
Why not use transform to transform from Number to boolean?

Reply:
Because we don't use the value. It's only there because I made the full schema.
I could but I feel that the only important thing here is for it to pass the schema validation.

Reply:
I think we can remove it then if we don't use it

Reply:
You should remove all values that we don't use. We don't need them. They just eat up network bandwidth and RAM

Reply:
IIRC we were talking multiple times about publishing our API controllers, I must have done it like so with this in mind.
The bandwidth usage is going to be the same no matter what but it is true that we could save up a little on processing power in this case.
If you both think I should remove it then I'll do so in this PR

Reply:
> I think we can remove it then if we don't use it
I agree. Remove it 👍